float64 latitude
float64 longitude
float32 altitude
float32 roll 
float32 pitch
float32 yaw